### PR TITLE
[Gekidou] Improve performance on find channels queries

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This PR makes some changes in the Query to get the Direct messages in the find channel screen and reduces the local search from ~3 seconds to < 100ms on android and from ~1s to < 50ms on iOS

Also changes the timing in the animations to improve on perceived perf

Note: This PR seems to work better than #6390 

```release-note
NONE
```

### CODE REVIEW

File: app/screens/find_channels/filtered_list/filtered_list.tsx [hunk 1]
```diff
@@ -244,12 +244,19 @@ const FilteredList = ({
...
+            items.push(...archivedAlpha.slice(0, MAX_RESULTS + 1));
+        }
+
         // Remote Channels that start with
         if (items.length < MAX_RESULTS) {
             items.push(...remoteChannels.startWith);
         }
 
-        // Users & Channels that matches
+        // // Users & Channels that matches
```
Comment (larkox) [inline app/screens/find_channels/filtered_list/filtered_list.tsx:259]: Unneeded change?

**Review by larkox — APPROVED** [review]
LGTM. If it does work better than my previous PR, we can remove close it

File: app/screens/find_channels/filtered_list/filtered_list.tsx (hunk 1 above)
Comment (enahum) [reply app/screens/find_channels/filtered_list/filtered_list.tsx:259]: @larkox currently without internet at home, would you mind removing the extra // :)

Thanks

Comment (larkox) [inline app/screens/find_channels/filtered_list/filtered_list.tsx:259]:
```suggestion
        // Users & Channels that matches
```

**Pull request merged**